cfme/ansible_tower/explorer.py

```diff
@@ -0,0 +1,144 @@
+
+    @View.nested
+    class tower_explorer_systems(Accordion):  # noqa
+        ACCORDION_NAME = 'Configured Systems'
+        tree = ManageIQTree()
+
+    @View.nested
+    class tower_explorer_job_templates(Accordion):  # noqa
+        ACCORDION_NAME = 'Job Templates'
+        tree = ManageIQTree()
+
+
+class TowerExplorerProviderToolbar(View):
+    reload = Button(title=VersionPick({Version.lowest(): 'Reload current display',
+                                       '5.9': 'Refresh this page'}))
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    download = Dropdown('Download')
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class TowerExplorerSystemJobTemplatesToolbar(View):
+    reload = Button(title=VersionPick({Version.lowest(): 'Reload current display',
+                                       '5.9': 'Refresh this page'}))
+    policy = Dropdown('Policy')
+    download = Dropdown('Download')
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class TowerExplorerView(BaseLoggedInPage):
+    search = View.nested(Search)
+    sidebar = View.nested(TowerExplorerAccordion)
+
+    @property
+    def in_tower_explorer(self):
+        return (self.logged_in_as_current_user and
+                self.navigation.currently_selected == ['Automation', 'Ansible Tower', 'Explorer'])
+
+
+class TowerExplorerProviderView(TowerExplorerView):
+    toolbar = View.nested(TowerExplorerProviderToolbar)
+
+
+class TowerExplorerSystemView(TowerExplorerView):
+    toolbar = View.nested(TowerExplorerSystemJobTemplatesToolbar)
+
+
+class TowerExplorerJobTemplatesView(TowerExplorerView):
+    toolbar = View.nested(TowerExplorerSystemJobTemplatesToolbar)
+
+
+class TowerExplorerProviderDefaultView(TowerExplorerProviderView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_tower_explorer and
+            self.title.text == 'All Ansible Tower Providers' and
+            self.sidebar.tower_explorer_providers.is_opened
+        )
+
+
+class TowerExplorerSystemDefaultView(TowerExplorerProviderView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_tower_explorer and
+            self.title.text == 'All Ansible Tower Configured Systems' and
+            self.sidebar.tower_explorer_systems.is_opened
+        )
+
+
+class TowerExplorerJobTemplatesDefaultView(TowerExplorerProviderView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_tower_explorer and
+            self.title.text == 'All Ansible Tower Job Templates' and
+            self.sidebar.tower_explorer_job_templates.is_opened
+        )
+
+
+class TowerExplorerProvider(Navigatable):
+    pass
+
+
+class TowerExplorerSystem(Navigatable):
+    pass
+
+
+class TowerExplorerJobTemplates(Navigatable):
+    pass
```

Comment:
**REQUIRED**
All these objects should have a respectful collection.

Reply:
I don't know why. I don't need collection for my tests and I would say same of others as nobody created collection for that until now. Also I have never worked with collections and I don't have time now to learn that...I just need to have these tests to have whole FA Automated.